README.md

```diff
@@ -53,72 +45,45 @@ So that I can stay constantly tapped in to the shouty box of Chitter
-
-
 Notes on functionality:
 ------
 
 * You don't have to be logged in to see the peeps.
 * Makers sign up to chitter with their email, password, name and a username (e.g. samm@makersacademy.com, password123, Sam Morgan, sjmog).
 * The username and email are unique.
 * Peeps (posts to chitter) have the name of the maker and their user handle.
-* Your README should indicate the technologies used, and give instructions on how to install and run the tests.
-
-Bonus:
------
-
-If you have time you can implement the following:
-
-* In order to start a conversation as a maker I want to reply to a peep from another maker.
 
-And/Or:
-
-* Work on the CSS to make it look good.
-
-Good luck and let the chitter begin!
-
-Code Review
------------
-
-In code review we'll be hoping to see:
-
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
+Getting started
+------
 
-Notes on test coverage
-----------------------
+1. Clone this repo 'git clone git@github.com:emanuelegorga/chitter-challenge.git'
+2. Change directory 'cd chitter-challenge'
+3. Install dependencies through 'bundle'
+4. Install postgresql through 'brew install postgres'
 
-Please ensure you have the following **AT THE TOP** of your spec_helper.rb in order to have test coverage stats generated
-on your pull request:
+Run the tests
+------
+1. Create a test database 'createdb chitter_test'
+2. Set up the test tables 'rake db:auto_migrate RACK_ENV=test'
+3. Run the tests 'rspec'
 
-```ruby
-require 'simplecov'
-require 'simplecov-console'
+How to use it
+------
+1. Create a development database 'createdb chitter_development'
+2. Set up the development tables 'rake db:auto_migrate'
+3. Run the server through 'rackup'
+4. Navigate [here](http://localhost:9292)
 
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
-  SimpleCov::Formatter::Console,
-  # Want a nice code coverage website? Uncomment this next line!
-  # SimpleCov::Formatter::HTMLFormatter
-])
-SimpleCov.start
-```
+The app
+------
+1. Sign Up
+2. Write your peeps
+3. You can see your peeps in your profile page or have a look at all peeps created by other users in the index.
 
-You can see your test coverage when you run your tests. If you want this in a graphical form, uncomment the `HTMLFormatter` line and see what happens!
+Technical Approach and technologies I used:
+-----
+* sinatra - To create the web app using Ruby;
+* postgresql - Database;
+* database_cleaner - Cleans up the database for each test ran;
+* data_mapper - Ruby ORM;
+* capybara - Test frameworks used for web developent.
```

Comment:
It's good to have the tech used here as you can refer back to it later - Consider adding any diagramming which may have happened to show your methodical workflow.